lib/easypost.rb

```diff
@@ -37,6 +37,9 @@
 require 'easypost/user'
 require 'easypost/webhook'
 
+#beta
```

Comment:
Let's remove the comment since the module already is `beta` and self explanatory

```suggestion
```

Reply:
removed